core/state_transition.go

```diff
@@ -43,7 +43,6 @@ The state transitioning model does all the necessary work to work out a valid ne
   4b) If valid, use result as code for the new state object
 == end ==
 5) Run Script section
-6) Derive new state root
```

Comment:
It is still done for pre-Byzantium blocks, so the comment is correct.

Reply:
As above